package.json

```diff
@@ -69,5 +69,6 @@
       "unlabeled": ":question: Unlabeled"
     },
     "wildcardLabel": "unlabeled"
-  }
-}
\ No newline at end of file
+  },
+  "packageManager": "pnpm@8.15.8+sha256.691fe176eea9a8a80df20e4976f3dfb44a04841ceb885638fe2a26174f81e65e"
```

Comment:
does this need to have the +sha... suffix? 🤔

Reply:
also the action.yaml removed version 8 restriction is that correct when this 'pins' it to 8?

Reply:
It works without the sha -- it's an integrity check to prevent installing of a MITM-injected version of the package manager